NitroxClient/GameLogic/PlayerPreferences/PlayerPreference.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.PlayerPreferences
+{
+    [Serializable]
+    public class PlayerPreference : IEquatable<PlayerPreference>
```

Comment:
Why not serialize and persist the Player object? We'll need to store that data anyways.

Reply:
Persistence of the player object seems like it should be maintained by the server rather than the client. That also feels like it is out-of-scope for this feature.

Reply:
This was my bad, I thought this was a server abstraction.